src/SprykerSdk/Yves/Benchmark/Dependency/Service/BenchmarkToUtilEncodingServiceBridge.php

```diff
@@ -0,0 +1,49 @@
+<?php
+
+/**
+ * Copyright © 2016-present Spryker Systems GmbH. All rights reserved.
+ * Use of this software requires acceptance of the Evaluation License Agreement. See LICENSE file.
+ */
+
+namespace SprykerSdk\Yves\Benchmark\Dependency\Service;
+
+class BenchmarkToUtilEncodingServiceBridge implements BenchmarkToUtilEncodingServiceInterface
+{
+    /**
+     * @var \Spryker\Service\UtilEncoding\UtilEncodingServiceInterface
+     */
+    protected $utilEncodingService;
+
+    /**
+     * @param \Spryker\Service\UtilEncoding\UtilEncodingServiceInterface $utilEncodingService
+     */
+    public function __construct($utilEncodingService)
+    {
+        $this->utilEncodingService = $utilEncodingService;
+    }
+
+    /**
+     * @param string $jsonValue
+     * @param bool $assoc
+     * @param int|null $depth
+     * @param int|null $options
+     *
+     * @return mixed|null
+     */
+    public function decodeJson($jsonValue, $assoc = false, $depth = null, $options = null)
+    {
+        return $this->utilEncodingService->decodeJson($jsonValue, $assoc, $depth, $options);
+    }
+
+    /**
+     * @param array $value
+     * @param int|null $options
+     * @param int|null $depth
+     *
+     * @return string|null
+     */
+    public function encodeJson($value, $options = null, $depth = null)
```

Comment:
Return type is missing.

Reply:
Original service interface has no return type